src/modules/HomePage/components/ProjectHistoryItem/ProjectHistoryItem.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div
+    class="bg-gray-300 rounded-lg px-4 py-2 m-2 flex items-center justify-start"
+    @click="handleClick()"
+  >
+    <svg
```

Comment:
Maybe search for a package to directly import svg files like `vue-cli-plugin-svg`